ams/dashboard/README.md

```diff
@@ -1 +1,92 @@
-# datalake-open
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one
+ - or more contributor license agreements.  See the NOTICE file
+ - distributed with this work for additional information
+ - regarding copyright ownership.  The ASF licenses this file
+ - to you under the Apache License, Version 2.0 (the
+ - "License"); you may not use this file except in compliance
+ - with the License.  You may obtain a copy of the License at
+ - 
+ -     http://www.apache.org/licenses/LICENSE-2.0
+ - 
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and 
+ - limitations under the License.
+-->
+
+## Requirements
+
+- node `16.18.0`
+- yarn `1.22.19`
+
+## Installation
```

Comment:
Perhaps we can organize the content according to this directory：
```
## Installing dependencies
## Running Locally
## Building project
```

This way may be clearer and make it easier for everyone to find the content they are interested in.

Reply:
Okay, I rearranged it according to the new table of contents.